cdk/lib/__snapshots__/stack.test.ts.snap

```diff
@@ -593,6 +593,7 @@ Object {
         "DBSubnetGroupName": Object {
           "Ref": "DatabaseSubnetGroup7D60F180",
         },
+        "DeleteAutomatedBackups": false,
         "DeletionProtection": true,
```

Comment:
Interesting that `DeletionProtection` is already `true` in the CFN diff - looks like it is [already being set](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_rds.DatabaseInstance.html#deletionprotection) due to:

```typescript
removalPolicy: RemovalPolicy.RETAIN
```

Reply:
Interesting! I'd like to keep the 'deletionProtection' explicit though (i.e. this PR as-is)